app/assets/stylesheets/views/_html-publication.scss

```diff
@@ -91,4 +91,17 @@
     }
   }
 
+  .back-to-content {
+    @include core-19;
+    display: block;
+    padding-bottom: $gutter-half;
+
+    &:before {
+      content: "\2191";
+    }
+
+    @media print {
```

Comment:
I'd like `@include dont-print;` mixin, and also have it exposed through `.dont-print`. Best of both worlds. :+1: